gnocchi/carbonara.py

```diff
@@ -407,6 +410,10 @@ def __init__(self, value, sampling):
             self.key = value.key
         elif isinstance(value, pandas.Timestamp):
             self.key = value.value / 10e8
+        elif isinstance(value, numpy.datetime64):
```

Comment:
what path would give splitkey a datetime64? having a little trouble figuring this out.